scripts/compile/compile-stage.ps1

```diff
@@ -30,7 +30,6 @@ $Projects = @(
     "Microsoft.DotNet.Tools.Resgen",
     "Microsoft.DotNet.Tools.Run",
     "Microsoft.DotNet.Tools.Test",
-    "Microsoft.Extensions.DependencyModel",
```

Comment:
No need to publish it directly, it will come wia compiler dependency anyway, avoids hitting "cannot publish to dnxcore50" error